v2/swiftui/App/Views/ItemList.swift

```diff
@@ -0,0 +1,25 @@
+import SwiftUI
+import RealmSwift
+
+/// View a list of all Items in the realm. User can swipe to delete Items.
+struct ItemList: View {
+    // ObservedResults is a collection of all Item objects in the realm.
+    // Deleting objects from the observed collection
+    // deletes them from the realm.
+    @ObservedResults(Item.self) var items
+    
+    var body: some View {
+        VStack {
+            List {
+                ForEach(items) { item in
+                    ItemRow(item: item)
+                }.onDelete(perform: $items.remove)
+            }
+            .listStyle(InsetListStyle())
```

Comment:
Format: this line should probably be on the previous line (connected to `}`) or indented 1 more (compare .frame below)

Reply:
I'll move .onDelete a line down, so it's consistent with the rest of the attributes and with Apple's examples.